### PR TITLE
fix parameters in clearPlaylist due to breaking changes in jsonrpc

### PR DESCRIPTION
**Pull request by csgitmeup**
one additional method needs parameters to be updated to fix the following issues:
https://github.com/home-assistant/core/issues/52731
https://github.com/home-assistant/core/issues/49856

Unfortunately I have limited possibilities to test the change. Works in my set-ups with the latest version of home-assistant docker and kodi 18.9 and 19.1

### CODE REVIEW

File: pykodi/kodi.py [hunk 1]
```diff
@@ -310,7 +310,7 @@ async def add_artist_to_playlist(self, artist_id):
 
     async def clear_playlist(self):
         """Clear default playlist (i.e. playlistid=0)."""
-        await self._server.Playlist.Clear({"playlistid": 0})
+        await self._server.Playlist.Clear(**{"playlistid": 0})
```
Comment (OnFreund) [inline pykodi/kodi.py:313]: Isn't this identical to:
```python
await self._server.Playlist.Clear(playlistid=0)
```
?

Comment (csgitmeup) [reply pykodi/kodi.py:313]: Changing it to your suggestion raises below error in home assistant for me.

`self._server.*(**{"wxy": z})` seems consistent with the rest of the calls  `"await self._server.*"` like lines 266, 289, 297, ... ?


---
```
[3954866736] invalid literal for int() with base 10: ''
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 185, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 1492, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1527, in _execute_service
    await handler.job.target(service_call)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 213, in handle_service
    await self.hass.helpers.service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 658, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 843, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 695, in _handle_entity_call
    await result
  File "/usr/src/homeassistant/homeassistant/components/kodi/media_player.py", line 256, in wrapper
    await func(obj, *args, **kwargs)
  File "/usr/src/homeassistant/homeassistant/components/kodi/media_player.py", line 704, in async_play_media
    await self.async_add_to_playlist(media_type_lower, media_id)
  File "/usr/src/homeassistant/homeassistant/components/kodi/media_player.py", line 772, in async_add_to_playlist
    await self._kodi.add_album_to_playlist(int(media_id))
ValueError: invalid literal for int() with base 10: ''
```

**Comment by OnFreund** [comment]
Thanks @csgitmeup. I released [version 0.2.7](https://pypi.org/project/pykodi/0.2.7/).

**Comment by OnFreund** [comment]
You can use this compare link to show the diff when you create the PR in HA:
https://github.com/OnFreund/PyKodi/compare/v0.2.6...v0.2.7

**Pull request merged**